pom.xml

```diff
@@ -6,11 +6,34 @@
     <groupId>com.amido.stacks.modules</groupId>
     <artifactId>stacks-modules-parent</artifactId>
     <version>1.0.0-SNAPSHOT</version>
-    <name>stacks-modules-parent</name>
+    <name>${project.groupId}:${project.artifactId}</name>
```

Comment:
Have you confirmed this concat has the same result as before?

Reply:
this the best practise to name a project , the concat  is different as the name include the group id as well